src/PgSql/Processor/TsVector.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Data\Db\PgSql\Processor;
```

Comment:
PostgreSQL-specific stuff should not end up in this repository. If we need arrays or JSON, we should implement these in db* package and its adpaters. Data-db only uses db.

Reply:
> 

I can moved it, no problem